### PR TITLE
[stable/testlink] Add missing tls secrets template

### PR DESCRIPTION
**Pull request by rafacamargo123**
Signed-off-by: Rafael de Camargo <rafaelcamargo@trt15.jus.br>

#### What this PR does / why we need it:

This chart was accepting "tls secrets" definition on its values, but the secret objects were not being created.

#### Which issue this PR fixes
none

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/testlink/templates/tls-secrets.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  labels:
+    app: "{{ template "testlink.fullname" $ }}"
+    chart: "{{ template "testlink.chart" $ }}"
+    release: {{ $.Release.Name | quote }}
+    heritage: {{ $.Release.Service | quote }}
+type: kubernetes.io/tls
+data:
+  tls.crt: {{ .certificate | b64enc }}
+  tls.key: {{ .key | b64enc }}
+---
```
Comment (miguelaeh) [inline stable/testlink/templates/tls-secrets.yaml]: Could you remove this line?

File: stable/testlink/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: testlink
-version: 6.2.1
+version: 6.2.2
```
Comment (miguelaeh) [inline stable/testlink/Chart.yaml]: This should be a minor version.

File: stable/testlink/values.yaml [hunk 3]
```diff
@@ -243,8 +243,14 @@ ingress:
   ## It is also possible to create and manage the certificates outside of this helm chart
   ## Please see README.md for more information
   # - name: testlink.local-tls
-  #   key:
-  #   certificate:
+  #   key: |-
+  #     -----BEGIN PRIVATE KEY-----
```
Comment (miguelaeh) [inline stable/testlink/values.yaml]: Why ` -----BEGIN PRIVATE KEY-----` and not `-----BEGIN RSA PRIVATE KEY-----` ?

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Thank you for the PR!!!
Please take a look to the comments.

File: stable/testlink/values.yaml (hunk 3 above)
Comment (rafacamargo123) [reply stable/testlink/values.yaml]: They are just different formats: https://tls.mbed.org/kb/cryptography/asn1-key-structures-in-der-and-pem
Personaly, I've never seen a `-----BEGIN RSA PRIVATE KEY-----`, all the certificates I ever bought were always `-----BEGIN PRIVATE KEY-----`, but no problem, I'll keep this comment like the others

File: stable/testlink/Chart.yaml (hunk 2 above)
Comment (rafacamargo123) [reply stable/testlink/Chart.yaml]: ou mean 6.2.1.1?

File: stable/testlink/templates/tls-secrets.yaml (hunk 1 above)
Comment (rafacamargo123) [reply stable/testlink/templates/tls-secrets.yaml]: sure

Comment (rafacamargo123) [reply stable/testlink/templates/tls-secrets.yaml]: wordpress helm files has this line too, that's why I kept it:

https://github.com/helm/charts/blob/c330fa62be204bba0c8e2050d4935b56ff27fd04/stable/wordpress/templates/tls-secrets.yaml#L14-L18

File: stable/testlink/Chart.yaml (hunk 2 above)
Comment (rafacamargo123) [reply stable/testlink/Chart.yaml]: ok

File: stable/testlink/templates/tls-secrets.yaml (hunk 1 above)
Comment (miguelaeh) [reply stable/testlink/templates/tls-secrets.yaml]: Yes, it should be deleted too.

**Review by miguelaeh — APPROVED** [review]
Thank you very much for addressing the changes!!

**Pull request merged**